.eslintrc.json

```diff
@@ -1,14 +1,16 @@
 {
   "extends": "eslint:recommended",
   "env": {
-    "browser": true
+    "browser": true,
+    "node": true,
```

Comment:
Don't think `node` should be on, since it might not flag things the examples. The `scripts` folder already has it one since those are nodejs scripts